$OpenBSD: patch-cmake_CMakeLists_txt,v 1.1 2013/12/08 19:13:34 zhuk Exp $
Make at least some paths really customizable, avoids CMake warnings.
--- cmake/CMakeLists.txt.orig	Sun Oct 13 17:20:23 2013
+++ cmake/CMakeLists.txt	Sun Oct 13 17:23:36 2013
@@ -1,5 +1,5 @@
-set(SMOKE_LIBRARY_PREFIX ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX})
-set(SMOKE_INCLUDE_DIR ${CMAKE_INSTALL_PREFIX}/include ${CMAKE_INSTALL_PREFIX}/include/smoke)
+set(SMOKE_LIBRARY_PREFIX ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX} CACHE PATH "Smoke library destination directory")
+set(SMOKE_INCLUDE_DIR ${CMAKE_INSTALL_PREFIX}/include ${CMAKE_INSTALL_PREFIX}/include/smoke CACHE PATH "Smoke headers installation directory")
 set(SMOKE_CMAKE_MODULE_DIR ${CMAKE_INSTALL_PREFIX}/share/smoke/cmake)
 set(SMOKE_GEN_BIN ${CMAKE_INSTALL_PREFIX}/bin/smokegen${CMAKE_EXECUTABLE_SUFFIX})
 set(SMOKE_GEN_SHARED ${CMAKE_INSTALL_PREFIX}/share/smokegen)
